packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -547,7 +564,14 @@ const MultiSelect = (props: Props) => {
         | React.ReactElement<React.ComponentProps<typeof SelectOpener>> => {
         const {noneSelected} = labels;
 
-        const menuContent = getMenuTextOrNode(allChildren);
+        const menuTextOrNode = getMenuTextOrNode(allChildren);
+        const [openerStringValue, openerContent] =
+            maybeExtractStringFromNode(menuTextOrNode);
```

Comment:
To simplify things, could we use the `getLabel` utility directly to get the announcement message? It looks like we updated `getMenuTextOrNode` and added `maybeExtractStringFromNode` to get the `openerStringValue`, which looks to always be the return value from `getLabel`! 

Let me know if I'm missing something though! It always takes me longer to understand the menu opener with all the flexibility it supports 😅

Reply:
I wish we could... unless I'm missing something, `getLabel` returns a node sometimes that is the wrong type for a live region (which requires a string). I needed a string label consistently, so that's why I had to `maybeExtractStringFromNode`.

Reply:
`getLabel` will always return a string 😄 
https://github.com/Khan/wonder-blocks/blob/78c353586dc24c9b9aed3b34d3d2a56f9ac85d2d/packages/wonder-blocks-dropdown/src/util/helpers.ts#L62-L72 

`getSelectOpenerLabel` returns a string or node!

Reply:
Oh yes, you're right! Drowning in complexity a bit here... I'll test it out! Simplification is always a good idea when we can do it.

Reply:
Ah, I remember now... it's the complexity from `getMenuTextOrNode` in `MultiSelect` that made this difficult, as there are multiple cases for selected items in a switch statement that return a string OR a node. I wanted to leverage that logic without having to iterate through it all again!

Reply:
Ahh gotcha, that makes sense!